test/conftest.py

```diff
@@ -24,6 +24,14 @@ def fixture_expected_block_hash(request):
     return request.param
 
 
+@pytest.fixture(name="dev_net_args")
+def fixture_dev_net_args(request):
```

Comment:
I'd define this just above the only test which is using it. Also, can you rename it to `starknet_wrapper_args`?